src/components/dialogs/parameters/dynamicsimulation/curve-parameters.tsx

```diff
@@ -154,7 +154,7 @@ const CurveParameters = ({ path }: { path: string }) => {
 
     return (
         <>
-            <Grid container direction={'column'} sx={{ height: 460 }}>
+            <Grid container direction={'column'} sx={{ height: 640 }}>
```

Comment:
Can we avoid this fixed height ?

Reply:
I have tried to fix it but not easy because the component TabPanel is shared among different params of calculations, see the PR which is forked from this PR: https://github.com/gridsuite/gridstudy-app/pull/2586